Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1526 @@
+                        "policy_name": policy_name,
+                        "amw_reputation": {
+                            "web_reputation": {"filtering": web_reputation_status},
+                            "adv_malware_protection": {
+                                "file_reputation_filtering": file_reputation_filtering_status,
+                                "file_reputation": {
+                                    file_reputation_action: [
+                                        "Known Malicious and High-Risk Files"
+                                    ]
+                                    if file_reputation_action
+                                    else None
+                                },
+                            },
+                            "cisco_dvs_amw": {
+                                "amw_scanning": {
+                                    "amw_scan_status": anti_malware_scanning_status
+                                },
+                                "suspect_user_agent_scanning": suspect_user_agent_scanning,
+                                "block_malware_categories": block_malware_categories,
+                                "block_other_categories": block_other_categories,
+                            },
+                        },
+                    }
+                ]
+            }
+        )
+
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+        )
+
+    def access_policy_delete_request(self, policy_names: str):
+        """
+        Delete access policy.
+
+        Args:
+            policy_names (str): Comma separated policy names to delete.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        params = assign_params(policy_names=",".join(policy_names))
+
+        return self._http_request(
+            "DELETE",
+            f"{V3_PREFIX}/web_security/access_policies",
+            params=params,
+            resp_type="response",
+        )
+
+    def domain_map_list_request(self) -> Dict[str, Any]:
+        """
+        List domain mappings.
+
+        Returns:
+            Dict[str, Any]: API response from Cisco WSA.
+        """
+        return self._http_request(
+            "GET", f"{V2_PREFIX}/configure/web_security/domain_map"
+        )
+
+    def domain_map_create_request(
+        self, domain_name: str, ip_addresses: List[str], order: Optional[int]
+    ) -> Dict[str, Any]:
+        """
+        Create domain mapping.
+
+        Args:
+            domain_name (str): Domain name.
+            ip_addresses (List[str]): IP addresses to map to the domain.
+            order (int): Index of domain map in the collection.
+
+        Returns:
+            Dict[str, Any]: API response from Cisco WSA.
+        """
+        data = [
+            {"IP_addresses": ip_addresses, "domain_name": domain_name, "order": order}
```

Comment:
```suggestion
           remove_empty_elements({"IP_addresses": ip_addresses, "domain_name": domain_name, "order": order})
```
What if order is None? 
🐶 + 🌊 = 🦭

Reply:
Order is required, I removed the Optional hint, it was a mistake